### ISSUE TITLE
Camera not Detecting Calibration Grid

### ISSUE THREAD

**Issue by scott40k** [bqlabs/horus#86]
Hi,

I have changed my camera from c310 to the correct c270 and still cannot get Horus to recognise the calibration grid at all, it seems to slow down the rotation when the grid side is shown but doesn't "see" it or overlay the coloured boxes...? Does anyone have any advice on this I have tried full un install and reinstall of the drivers for the webcam.

**Comment by witkop** [comment]
This may seem a silly comment, but as a newby to 3d scanning, it fooled me for a while. When I printed the calibration grid, I made the assumption that it was on an A4 size sheet. When I used this size to do the calibration scan it failed to recognise the grid properly. After a lot of toil I inally noticed that the grid was a lot smaller than the A4 size sheet and reprinted it with approx. 13mm squares. This then worked correctly. The other thing I found was that I had to 'tweak' the number of rows and columns slightly up/down to get recognition. Hope this helps.
